### DISCUSSION TITLE
error in flash wifi example:

### DISCUSSION THREAD

**Discussion by thebikerholic** [bitcraze/discussions#117] [Q&A]
cfloader flash aideck_gap8_wifi_img_streamer_with_ap.bin deck-bcAI:gap8-fw -w radio://0/80/2M
Reset to bootloader mode ...
Failed to flash: Could not connect to bootloader

while implementing thios code in the terminal i got the error like this please help to resolve the proble
![Screenshot from 2022-06-28 15-23-50](https://user-images.githubusercontent.com/101856934/176150545-ae0d443c-fd87-45c3-8126-51d981ccbc36.png)
m.

**Comment by knmcguire** [comment]
Hi! Some more questions to your problem:
- Could check on the bottom which revision of the aideck 1.1 you have? (Rev.C or .D)
- Have gone fully through the [getting started tutorial](https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/)? It has been updated with some more trouble shooting notes.

**Reply by thebikerholic** [reply]
yes i have go through the all the tutorial but even i got an error my AI deck is REV C version.

**Reply by knmcguire** [reply]
Ah oke, that means that the [ GAP8 bootloader](https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/#gap8-bootloader-for-older-aidecks-revisions-a-c-) should have been manually installed then. 

But this error might not be related to the aideck actually, but the inability of cfloader to connect to your crazyflie:
- Do you know the URI of your Crazyflie? It's supposed to be in the form of radio://0/80/2M/E7E7E7E7E7. You can check it with connecting a crazyflie with a USB and check the config: https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/#firmware-configuration

**Comment by thebikerholic** [comment]
Thank you i can check that and rectify it.
